compliance/fnms/fnms_low_licenses_available/README.md

```diff
@@ -0,0 +1,62 @@
+# FlexNet Manager Low Available Licenses - Cloud
+
+## What it does
+
+This policy uses a FlexNet Manger Cloud and looks up all FlexNet Manager Licenses that has lower than user provided percentage.
+
+The report / Mail output looks like this:
+![Alt text][emailoutput]
+
+Current limitations:
+
+- Output is limited to max 100000 rows.
+
+## Prerequisites
+
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
+
+### Credential configuration
+
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
+
+Credential type must be:
+
+- *API Key* - For FNMS Cloud  -  (*Location*:header, *Type*: Bearer)
+- *NTLM* - For OnPrem FNMS
```

Comment:
The provider value isn't listed.  The user needs to know the provider tag so the UI knows which Credentials to show.

Reply:
Also OnPrem should be On Premise.